loader/terminator.c

```diff
@@ -108,6 +110,7 @@ VKAPI_ATTR void VKAPI_CALL terminator_GetPhysicalDeviceSparseImageFormatProperti
                                                                         tiling, pNumProperties, pProperties);
     }
 }
+#endif // VULKANSC
```

Comment:
Why not create an SC and a non-SC version of this terminator file?  Or at least define the non-SC fallback paths in an inlined utility function for each function that can be empt and ignored for SC?  Then the code would be cleaner and easier to follow.